script/bootstrap.sh

```diff
@@ -17,19 +17,23 @@ setup_rbenv() {
   export PATH="$HOME/.rbenv/bin:$PATH"
   eval "$(rbenv init -)"
 
-  if [ ! -d "$HOME/.rbenv/versions/2.3.1" ]; then
-    subtask_exec "Install ruby 2.3.1" rbenv install 2.3.1
+  brew install rbenv
+
+  if [ ! -d "$HOME/.rbenv/versions/2.4.2" ]; then
+    subtask_exec "Install ruby 2.4.2" rbenv install 2.4.2
   fi
 
   if [ ! -f "$HOME/.rbenv/version" ]; then
-    subtask_exec "Setting global ruby version to 2.3.1" rbenv global 2.3.1
+    subtask_exec "Setting global ruby version to 2.4.2" rbenv global 2.4.2
   fi
 }
 
 setup_postgresql() {
   task_inform "Setting up PostgreSQL"
   export PATH="/usr/local/opt/postgresql@9.6/bin:$PATH" # This is necessary b/c the postgresql binaries aren't yet in the user's $PATH
 
+  brew install postgresql@9.6
```

Comment:
Same here, this could go into a Brewfile. At the very least, you probably probably don't want to export the PATH change until you confirm that postgresql is installed.